CHANGELOG.md

```diff
@@ -15,7 +16,6 @@ FEATURES
 IMPROVEMENTS
 
 * licensing (Consul Enterprise): Increase initial server temporary license duration to 6 hours to allow for longer upgrades/migrations.
-* agent: some check types now support configuring a number of consecutive failure and success before the check status is updated in the catalog. [[GH-5739](https://github.com/hashicorp/consul/pull/5739)]
```

Comment:
I see this isn't in v1.6.2 tag but what happened to it? Should it be moved to UNRELEASED section? I see it's still in `master`. Is there a chance it'll make it in `v1.6.3`? Sorry if I missed some discussion already with this.

Reply:
@alvin-huang can give the full explanation but master is the next major release, essentially – and we cut from the `release/1.6.x` and need to port over the changelog update.

I believe we're investigating an automated commit to make this faster after release so users browsing master aren't surprised.

Reply:
Hi @fastest963 that PR didn't make it into 1.6.2 but will make it into the next release of Consul. This may be 1.7.0 or 1.6.3 depending on if we feel there is a need for an incremental release of the current version.

Reply:
Thanks for the clarification!